test/smoke/package.json

```diff
@@ -18,12 +15,9 @@
     "@types/mocha": "2.2.41",
     "@types/ncp": "2.0.1",
     "@types/node": "^10.14.8",
-    "@types/puppeteer": "^1.19.0",
```

Comment:
Just realized puppeteer is in the root package.json, it should probably live in smoke/ (and automation/ after this PR), created https://github.com/microsoft/vscode/issues/80329.

Reply:
Yes, I had wondered about that. I see you already made the change. I rebased on top of that and then moved the dependency over to the `automation` package.